packaging/nnfw.spec

```diff
@@ -172,12 +173,7 @@ tar -xf %{SOURCE3016} -C ./runtime/externals
 
 %if %{odc_build} == 1
 mkdir ./externals
-tar -xf %{SOURCE3004} -C ./externals
-tar -xf %{SOURCE3005} -C ./externals
-tar -xf %{SOURCE3008} -C ./externals
-tar -xf %{SOURCE3013} -C ./externals
-tar -xf %{SOURCE3014} -C ./externals
-tar -xf %{SOURCE3015} -C ./externals
```

Comment:
How can the combined tar ball be less than 2MB when the original tensorflow tar ball size is over 7MB?

Reply:
Because `luci-compute` requires subset code files from tensorflow and gemmlowp.